crates/ensure_no_std/Cargo.toml

```diff
@@ -0,0 +1,25 @@
+[package]
+name = "ensure_no_std"
+version = "0.7.0"
+authors = ["Supercomputing Systems AG <info@scs.ch>"]
+license = "Apache-2.0"
+edition = "2021"
+
+[dependencies]
+libc = { version = "0.2.119", default-features = false }
+
+nostr = { path = "../nostr", default-features = false, features = ["alloc"] }
+
+#secp256k1 = { version = "0.24", default-features = false, features = ["alloc", "serde", "rand"]}
+
+#rand = { version = "0.8", features = [
+#    "alloc",
+#    "getrandom",
+#], default-features = false }
+
+#url_no_std = { package = "url", features = [
+#    "idna",
+#    "alloc",
+#    "core-error",
+#    "core-net",
+#], path = "../../../rust-url/url", default-features = false }
```

Comment:
Are these still needed?

Reply:
They are not, removed in https://github.com/integritee-network/nostr/pull/1/commits/1925dae6668812b41ef5c8b0556b9b39ae4944ea.